tooling/template-base/pages/index/index.page.tsx

```diff
@@ -7,13 +7,13 @@ import CallToActionWithLinks from '../../components/CTA/CallToActionWithLinks';
 import CallToActionParagraphs from '../../components/CTA/CallToActionParagraphs';
 import NavBar from '../../components/Navigation/NavBar';
 import { HydratedContentItem } from '@thoughtindustries/content';
+import { getPageMeta } from '../../renderer/getPageMeta';
 
 export { Page };
 export { documentProps };
 
 const documentProps = {
-  title: 'Home Page',
-  description: 'The home page'
+  getPageMeta
```

Comment:
Don't think this is the right use of `getPageMeta`. `getPageMeta` is for getting the defined `documentProps` in this file, so this wouldn't necessarily work and should be reverted to its prior value.